concepts/40 Angular Components/40 Common Features/15 Intercept HTTP Requests.md

```diff
@@ -0,0 +1,32 @@
+To intercept Ajax requests in DevExtreme Angular components like [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/), [FileUploader](/Documentation/ApiReference/UI_Components/dxFileUploader/), and [Map](/Documentation/ApiReference/UI_Components/dxMap/) with Angular [HttpInterceptor](https://angular.io/api/common/http/HttpInterceptor), import additional `DxHttpModule`.
```

Comment:
This is a pretty long sentence - many things are going on. 

- The verb "intercept" and the means of interception ("HttpInterceptor") are too far from each other. I'd rather have the component list at the end of the sentence. 
- You mention "Angular" two times and I guess we can find a way to indicate it's all about Angular once. 
- We can split the sentence into two. 

So here's my suggestion: 

```suggestion
In Angular apps, you can use [HttpInterceptor](https://angular.io/api/common/http/HttpInterceptor) to intercept Ajax requests in DevExtreme components such as [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/), [FileUploader](/Documentation/ApiReference/UI_Components/dxFileUploader/), and [Map](/Documentation/ApiReference/UI_Components/dxMap/). This task requires that you import `DxHttpModule`.
```